src/components/v2/Layout/Sidebar/styles.ts

```diff
@@ -0,0 +1,86 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useStyles = ({ expanded }: { expanded: boolean }) => {
+  const theme = useTheme();
+  return {
+    drawer: css`
+      display: none;
+      ${theme.breakpoints.up('md')} {
```

Comment:
Since we're building the website as desktop first app, we need to use theme.breakpoints.down instead.